esphome/components/cd74hc4067/sensor.py

```diff
@@ -0,0 +1,55 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import sensor, voltage_sampler
+from esphome.const import (
+    CONF_ID,
+    CONF_SENSOR,
+    CONF_NAME,
+    CONF_NUMBER,
+    ICON_FLASH,
+    UNIT_WATT,
+    STATE_CLASS_MEASUREMENT,
+    DEVICE_CLASS_POWER,
+)
+from . import cd74hc4067_ns, CD74HC4067Component
+
+DEPENDENCIES = ["cd74hc4067"]
+
+CD74HC4067Sensor = cd74hc4067_ns.class_(
+    "CD74HC4067Sensor",
+    sensor.Sensor,
+    cg.PollingComponent,
+    voltage_sampler.VoltageSampler,
+)
+
+CONF_cd74hc4067_ID = "cd74hc4067_id"
```

Comment:
```suggestion
CONF_CD74HC4067_ID = "cd74hc4067_id"
```

Those CONF_ names are always uppercase in esphome

Reply:
changed